package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@langleyfoxall/react-dynamic-data-table",
-  "version": "4.0.0",
+  "version": "5.0.0",
```

Comment:
I wouldn't say this is a breaking change? Why not `4.1.0`?

Reply:
This is breaking, because it changes the default `axios` instance used from `require('axios'),` to `window.axios || require('axios')`.